tests/integration_tests/test_applications.py

```diff
@@ -559,9 +559,57 @@
             "--submit_command",
             "local",
             "--data_directory",
-            "TESTMODELDIR/",
+            "TESTOUTPUTDIR/",
             "--output_path",
-            "TESTMODELDIR/",
+            "TESTOUTPUTDIR/",
+        ]
+    ],
+    "simulate_prod::gamma_20_deg": [
+        [
+            "--production_config",
+            "./tests/resources/prod_multi_config_test.yml",
+            "--model_version",
+            "Prod5",
+            "--site",
+            "north",
+            "--primary",
+            "gamma",
+            "--azimuth_angle",
+            "north",
+            "--zenith_angle",
+            "20",
+            "--start_run",
+            "0",
+            "--run",
+            "1",
+            "--nshow",
+            "10",
+            "--data_directory",
+            "TESTOUTPUTDIR/",
+        ]
+    ],
+    "simulate_prod::gamma_20_deg_pack_for_grid": [
```

Comment:
I don't see what is different in simulate_prod::gamma_20_deg_pack_for_grid compared to "simulate_prod::gamma_20_deg. It is only the `--run` option - does this need to be tested?

Reply:
You are right, while changing options back and forth I forgot to put back the `"--pack_for_grid_register",` option for the second test. I added it now.

Reply:
Good!